app/views/shared/_osd_div.html.slim

```diff
@@ -9,6 +9,8 @@
   a#osd_zoom_out =svg_symbol "#icon-zoom-out", class: 'icon'
   a#osd_home =svg_symbol "#icon-fit-view", class: 'icon'
   a#osd_fit_width(title="#{t('.fit_width')}") =svg_symbol "#icon-fit-width", class: 'icon'
+  -if !@is_monitor_view
+    a#osd_monitor(title="#{t('.monitor')}") =svg_symbol "#icon-window", class: 'icon'
```

Comment:
Should we replace this with a "close window" icon when the monitor view is being displayed?

Reply:
yeah, that would be great and what I had been thinking of. 
I will try to change the icon using Javascript.